using_the_compiler/README.md

```diff
@@ -12,14 +12,14 @@ To compile and run a program in a single shot you can invoke `crystal` with a si
 $ crystal some_program.cr
 ```
 
-Crystal files end with the `.cr` extension.
```

Comment:
I agree that this reads weird, but I'm not sure the change fixes that. What do you think about moving the "Crystal files end with the `.cr` extension" sentence a bit upwards so that the previous sentence reads:

"To compile and run a program in a single shot, you can invoke `crystal` with a single filename (Crystal files end with `.cr`): (...)"